include/bbcode.php

```diff
@@ -238,35 +241,48 @@ function bb_find_open_close($s, $open, $close, $occurance = 1) {
 
 function get_bb_tag_pos($s, $name, $occurance = 1) {
 
-	if($occurance < 1)
+	if ($occurance < 1)
 		$occurance = 1;
 
 	$start_open = -1;
-	for($i = 1; $i <= $occurance; $i++) {
-		if( $start_open !== false)
+	for ($i = 1; $i <= $occurance; $i++) {
+		if ( $start_open !== false) {
 			$start_open = strpos($s, '[' . $name, $start_open + 1); // allow [name= type tags
+		}
 	}
 
-	if( $start_open === false)
+	if ( $start_open === false) {
 		return false;
+	}
 
 	$start_equal = strpos($s, '=', $start_open);
 	$start_close = strpos($s, ']', $start_open);
 
-	if( $start_close === false)
+	if ( $start_close === false) {
```

Comment:
17 more occurrences found. Oh, occurence is red-lined, occur**r**ence not. Maybe U.S. way of the word? I know British English is different to American English.